OpenRA.Mods.Common/Widgets/ColorMixerWidget.cs

```diff
@@ -32,6 +32,7 @@ public class ColorMixerWidget : Widget
 
 		bool update;
 		object syncWorker = new object();
+		object frontWorker = new object();
```

Comment:
I would rename this, maybe ```bufferLock``` or ```bufferSync``` but I'm open to other suggestions. ```frontWorker``` makes little sense though.